massa-execution-worker/src/execution_info.rs

```diff
@@ -0,0 +1,74 @@
+#![allow(dead_code)]
+
+use std::collections::HashMap;
+
+use crate::execution::ExecutionResult;
+use massa_models::address::Address;
+use massa_models::amount::Amount;
+use massa_models::slot::Slot;
+
+pub struct ExecutionInfo {}
+
+impl ExecutionInfo {
+    pub(crate) fn new() -> Self {
+        Self {}
+    }
+}
+
+pub enum OperationInfo {
+    RollBuy(u64),
+    RollSell(u64),
+}
+
+pub struct ExecutionInfoForSlot {
+    pub(crate) block_producer_reward: Option<(Address, Amount)>,
```

Comment:
I don't understand why there is less fields in the structure `ExecutionOutput` of `massa-execution-exports` than here. Moreover it's not possible to have common structure defined in `massa-execution-exports` and used in this module ?

Reply:
In fact, I forgot to add ExecutionInfoForSlot into ExecutionInfo (which is in ExecutionOutput) but I've just fix this...